packages/shared/components/fund-withdraw/fund-withdraw-wallet.tsx

```diff
@@ -0,0 +1,51 @@
+import { WalletData } from "gv-api-web";
+import React, { Component } from "react";
+import { InjectedTranslateProps, translate } from "react-i18next";
+import WalletImage from "shared/components/avatar/wallet-image/wallet-image";
+import Select from "shared/components/select/select";
+
+interface IFundWithdrawWalletProps {
+  wallets: WalletData[];
+  value: string;
+  onChange(value: string): void;
+}
+
+class FundWithdrawWallet extends Component<
+  IFundWithdrawWalletProps & InjectedTranslateProps
+> {
+  handleChange = (event: any) => {
```

Comment:
I thing select change have own event like `SelectEvent` or similar. Maybe suitable for use type `Event`